ansible/library/load_extra_dpu_config.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python
+
+import paramiko
+import os
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.smartswitch_utils import smartswitch_hwsku_config
+
+DPU_HOST_IP_BASE = "169.254.200.{}"
+SRC_DPU_CONFIG_FILE = "/tmp/dpu_extra.json"
+DST_DPU_CONFIG_FILE = "/tmp/dpu_extra.json"
+CONFIG_LOAD_CMD = "sudo config load {} -y".format(DST_DPU_CONFIG_FILE)
```

Comment:
Need to change to reload. Since OA has depdency on DEVICE_METADATA table. Several container need to restart.